src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs

```diff
@@ -45,7 +45,7 @@
 // to distinguish one build from another. AssemblyFileVersion is specified
 // in AssemblyVersionInfo.cs so that it can be easily incremented by the
 // automated build process.
-[assembly: AssemblyVersion("2.13.0.2104")]
+[assembly: AssemblyVersion("2.13.0.2305")]
```

Comment:
Correct me if I am wrong but I don't think we normally submit changes to these version numbers in a typical PR.

Reply:
I do not think anyone is actually using the build number for anything.
It gets incremented on every Build automatically and I sometimes commit it by mistake.

I reverted it.